### ISSUE TITLE
Rename branch "master" to "main"

### ISSUE THREAD

**Issue by ameowlia** [cloudfoundry/routing-release#171]
### Current State
We have 3 non-wip branches that we use for routing-release. 

| **branch name**   | **purpose**  |
|---|---|
| develop  |  Where we push code we are actively developing. When people submit PRs, they submit them against the  develop branch.  |
|  release-candidate | Once the code on develop passes our pipeline, it is merged into this branch automatically.  |
|  master |  When we release, we merge release-candidate into the master branch. |

### Desired State

* Replace "master" with "main"

### Why does this matter?

Master/slave terminology has existed for decades and has [permeated many aspects of computer science and other fields](https://en.wikipedia.org/wiki/Master/slave_(technology)). [It is also extremely racist](https://cdm.link/2020/06/lets-dump-master-slave-terms/). We should stop using these terms. 

There has been some debate on the [cf-dev mailing list](https://lists.cloudfoundry.org/g/cf-dev/messages?expanded=1&msgnum=9077) about whether teams should replace "master" with "main" or with some other term. I think we should chose "main" because that is what [github changing its default to](https://www.vice.com/en_us/article/k7qbyv/github-to-remove-masterslave-terminology-from-its-platform). By choosing the same default branch name as github this will provide predictability and cohesion between repos.

**Comment by mcwumbly** [comment]
I'm in favor of "main" if we believe that it'll be more consistent with other repos.

If consistency seems unattainable, then I'd vote for "release", given the table you helpfully put together above.  Let's include that table (with the new name) in https://github.com/cloudfoundry/routing-release#developer-workflow and/or in a new CONTRIBUTING.md while we're at it!

**Comment by kauana** [comment]
Done! We renamed the master branch to release: https://github.com/cloudfoundry/routing-release/commit/9f9f04b0ac952b824ec08073f27080bb6a75e4dc
